src/containerapp/azext_containerapp/_params.py

```diff
@@ -117,6 +117,12 @@ def load_arguments(self, _):
         c.argument('environment_name', options_list=['--environment'], help="The environment name.")
         c.argument('resource_group_name', arg_type=resource_group_name_type, id_part=None)
 
+    with self.argument_context('containerapp env', arg_group='Custom Domain') as c:
+        c.argument('certificate_identity', options_list=['--custom-domain-certificate-identity', '--certificate-identity'], 
+                   help='Resource ID of a managed identity to authenticate with Azure Key Vault, or System to use a system-assigned identity.', is_preview=True)
+        c.argument('certificate_key_vault_url', options_list=['--custom-domain-certificate-key-vault-url', '--certificate-key-vault-url'], 
```

Comment:
>'--custom-domain-certificate-key-vault-url', '--certificate-key-vault-url' [](http://example.com/codeflow?start=62&length=74)

--certificate-identity and -k?

Reply:
![image](https://github.com/Azure/azure-cli-extensions/assets/2786738/bf0e23e4-5f76-4fb9-9530-e03d478b325a)
just follow the same format as existing custom domain parameters